apps/web/src/store/reducers/ormReducer/index.js

```diff
@@ -776,6 +777,88 @@ export default function ormReducer (state = orm.getEmptyState(), action) {
       break
     }
 
+    case UPDATE_MEMBERSHIP_NAV_ORDER_PENDING: {
+      me = Me.first()
+      membership = Membership.safeGet({ group: meta.groupId, person: me.id })
+
+      if (!membership) break
+
+      const newNavOrder = meta.navOrder
```

Comment:
Would be nice to bundle this into a platform agnostic function that can live in packages/shared. Otherwise we'll be repeating the operation in three spots

Reminds me that I should move reorderTree over to shared

Reply:
I went through that process with context widgets; separating out the logic from the platform/environment specific side-effects. Overall this logic should be simpler because its just a flat list ordering... but if we hit bugs a bunch in this, it might be worth coming back to it and pulling the logic out into a directly testable function